pkg/bpf/detect.go

```diff
@@ -42,12 +42,10 @@ func detectOverrideHelper() bool {
 		},
 		License: "GPL",
 	})
-	prog.Close()
```

Comment:
that's the kind of situation where you can use `defer`:
```go
defer prog.Close()
```
so that's it is executed anyway.

Reply:
hum, but we need to be sure the program got loaded and then we just return, so does not seem defer is helpful in this case.. but I just found same problem in detectLargeProgramSize, will fix that

Reply:
oh okay nevermind!!